docs/content/en/docs/Coding Milestones/PoC2023q1/edge-syncer.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Edge Syncer"
+date: 2023-03-22
+weight: 4
+description: >
+---
+
+{{% pageinfo %}}
+Edge syncer is a new syncer working with mailbox workspace in edge-mc. 
+{{% /pageinfo %}}
+
+![edge-syncer drawio](./images/edge-syncer-overview.png)
+
+#### Registering Edge Syncer on a p-cluster
+
+Edge-syncer can be deployed on p-cluster easily by the following steps.
+1. Create SyncTarget and Location
+  - Mailbox controller creates mailbox workspace automatically. 
+  - Get mailbox workspace id
+2. Use command to register edge-syncer
+   ```console
+   kubectl ws <mb-ws name>
+   kcp workload edge-sync <EM Sync Target name> --image <EM Syncer Image> -o edge-syncer.yaml
```

Comment:
I am not sure what you mean by the `kcp` command here.  kcp-dev/kcp/cmd/kcp does not have a "workspace" subcommand.  I expect this is a typo for `kubectl kcp`?

Reply:
Yes, this is typo. thank you!